docs/resources/execute.md

```diff
@@ -148,4 +123,6 @@ resource "snowflake_execute" "test" {
 
 Import is supported using the following syntax:
 
-{{codefile "shell" "/Users/jcieslak/Documents/terraform-provider-snowflake/examples/resources/snowflake_execute/import.sh"}}
+```shell
+terraform import snowflake_execute.example '<random-uuid>'
```

Comment:
Why not the usual file embedding?

Reply:
Hmm it's done the usual way in the template file